test/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.t.sol

```diff
@@ -0,0 +1,1036 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "forge-std/console.sol";
+
+// SuT
+import {
+    IBancorVirtualSupplyBondingCurveFundingManager,
+    BancorVirtualSupplyBondingCurveFundingManager,
+    IFundingManager
+} from
+    "src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol";
+
+// External Libraries
+import {Clones} from "@oz/proxy/Clones.sol";
+
+// Internal Dependencies
+import {ModuleTest, IModule, IOrchestrator} from "test/modules/ModuleTest.sol";
+import {BancorFormula} from
+    "src/modules/fundingManager/bondingCurveFundingManager/formula/BancorFormula.sol";
+import {IVirtualTokenSupply} from
+    "src/modules/fundingManager/bondingCurveFundingManager/marketMaker/IVirtualTokenSupply.sol";
+import {IVirtualCollateralSupply} from
+    "src/modules/fundingManager/bondingCurveFundingManager/marketMaker/IVirtualCollateralSupply.sol";
+import {
+    IRedeemingBondingCurveFundingManagerBase,
+    IRedeemingBondingCurveFundingManagerBase
+} from
+    "src/modules/fundingManager/bondingCurveFundingManager/RedeemingBondingCurveFundingManagerBase.sol";
+// Errors
+import {OZErrors} from "test/utils/errors/OZErrors.sol";
+
+// Mocks
+import {BancorVirtualSupplyBondingCurveFundingManagerMock} from
+    "test/modules/fundingManager/bondingCurveFundingManager/marketMaker/utils/mocks/BancorVirtualSupplyBondingCurveFundingManagerMock.sol";
+
+import {RedeemingBondingCurveFundingManagerBaseTest} from
+    "test/modules/fundingManager/bondingCurveFundingManager/RedeemingB_CurveFundingManagerBase.t.sol";
+
+/*     
+    @NOTE: The functions:
+
+    - deposit(uint amount) external {}
+    - depositFor(address to, uint amount) external {}
+    - withdraw(uint amount) external {}
+    - withdrawTo(address to, uint amount) external {} 
+
+    are not tested since they are empty and will be removed in the future.
+
+    Also, since the following functions just wrap the formula contract, their content is assumed to be tested in the formula tests, not here:
```

Comment:
Currently there arent any formular tests

Reply:
Because we are taking the formula wholesale from Aragon, so we don't re-test it. Updated the wording